sys-cluster/teleport/teleport-2.0.6.ebuild

```diff
@@ -0,0 +1,65 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+inherit eutils golang-build systemd user
+
+DESCRIPTION="Modern SSH server for teams managing distributed infrastructure"
+HOMEPAGE="https://gravitational.com/teleport"
+
+EGO_PN="github.com/gravitational/${PN}/..."
+
+if [ ${PV} == "9999" ] ; then
+	inherit git-r3 golang-vcs
+	EGIT_REPO_URI="https://github.com/gravitational/${PN}.git"
+else
+	inherit golang-vcs-snapshot
+	SRC_URI="https://github.com/gravitational/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64"
+fi
+
+LICENSE="Apache-2.0"
+SLOT="0"
+IUSE="etcd systemd"
+
+DEPEND="
+	>=dev-lang/go-1.7
+	app-arch/zip"
+RDEPEND="
+	systemd? ( sys-apps/systemd )
+	etcd? ( dev-db/etcd )"
+
+src_prepare() {
+	default
+
+	for i in "${FILESDIR}"/*;do cp "$i" "${S}";done
+
+	if use etcd; then
```

Comment:
To be honest, I don't like the idea of using patches to alter defaults like this. I think it'd be better to make sure you can easily sed them in. Or even remove this completely, USE flags are not supposed to be used to edit configuration files.